Examples/Algorithms/Geant4/include/ActsExamples/Geant4/Geant4Options.hpp

```diff
@@ -0,0 +1,66 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2017 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "ACTFW/Utilities/Options.hpp"
+#include "Acts/Utilities/Logger.hpp"
+#include "Acts/Utilities/Units.hpp"
+
+#include <iostream>
+
+#include "GeantinoRecording.hpp"
+
+namespace po = boost::program_options;
+using namespace Acts::UnitLiterals;
+
+namespace FW {
+
+namespace Options {
+
+/// @brief ExtrapolationAlgorithm options
+///
+/// @tparam aopt_t Type of the options class from boost
+template <typename aopt_t>
+void addGeant4Options(aopt_t& opt) {
```

Comment:
This does not need to be templated. The implementation can be placed in a `.cpp` file. The argument should be of type `Options::Description` defined e.g. in `ACTFW/Utilities/OptionsFwd.hpp` to avoid having to include the full `boost::program_options` in the header.